packages/publisher/README.md

```diff
@@ -408,32 +351,12 @@ Optional. Example value `latest`
 
 If present, packages are published with the provided version tag.
 
-### azureStorageAccount
```

Comment:
so...we can drop our azure storage account too?

Reply:
I believe it’s no longer used, but could be useful in the future (we can automatically send application logs there, for example)